### PR TITLE
Sets up RSpec for unit testing

### PR DESCRIPTION
**Pull request by CamronLDNF**
## Pivotal Tracker link 
[Link to PT](https://www.pivotaltracker.com/story/show/162085464)
## Gems installed
-  'factory_bot_rails'
- 'pry-rails'
- 'rspec-rails'
- 'shoulda-matchers'
## Actions taken
- Installed RSpec
- Modified spec/rails_helper.rb, config/application.rb
## What we have learned
- How to set up RSpec for unit testing

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# This file was generated by the `rails generate rspec:install` command. Conventionally, all
```
Comment (oliverochman) [inline spec/spec_helper.rb]: Clean up this file

**Review by oliverochman — CHANGES_REQUESTED** [review]
Pull in the upstream development, clean up the spec_helper and then we can merge it

File: spec/spec_helper.rb (hunk 1 above)
Comment (CamronLDNF) [reply spec/spec_helper.rb]: Cleaned up now.

**Comment by CamronLDNF** [comment]
We have cleaned out the comments and resolved merge conflicts.

File: spec/spec_helper.rb (hunk 1 above)
Comment (tochman) [reply spec/spec_helper.rb]: Not what I can see. Still need to get rid of the comments.

Comment (CamronLDNF) [reply spec/spec_helper.rb]: Sorry, wrong file cleaned. Done now, promise.

**Review by oliverochman — APPROVED** [review]

**Review by oliverochman — APPROVED** [review]
LGTM! Next PR I want to see some git ponging

**Pull request merged**